Loop/Views/BolusProgressTableViewCell.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  BolusProgressTableViewCell.swift
+//  LoopUI
+//
+//  Created by Pete Schwamb on 3/11/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+import HealthKit
+import MKRingProgressView
+
+public class BolusProgressTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var progressLabel: UILabel!
+
+    @IBOutlet weak var stopSquare: UIView! {
+        didSet {
+            stopSquare.layer.cornerRadius = 3
```

Comment:
I think this corner radius is _slightly_ too large. I played briefly with changing it and found that a value of 2 preserves the square shape more clearly.